DEPS

```diff
@@ -31,7 +31,7 @@ vars = {
   # Dart is: https://github.com/dart-lang/sdk/blob/master/DEPS.
   # You can use //tools/dart/create_updated_flutter_deps.py to produce
   # updated revision list of existing dependencies.
-  'dart_revision': '1aaebedd7045a44ba92bb1bb27c3a8bc25d74856',
+  'dart_revision': 'ee15c8eb689791f6777eb3f6b0a1e9c58ff0671b',
```

Comment:
Consider rolling dart-sdk separately (or at least add a note about dart roll into commit message).

Reply:
Yes, my intent is to merge this away once dart rolls.